yt/visualization/particle_plots.py

```diff
@@ -438,20 +440,28 @@ def __init__(
             )
 
             oblique = True
+            periodic = False
+            if origin != "center-window":
+                mylog.warning(
+                    "The 'origin' keyword is ignored for off-axis "
+                    "particle projections, it is always 'center-window'"
+                )
```

Comment:
shouldn't this be a actual warning (`warnings.warn`) ? My rule of thumb is that `mylog.warning` should only be used for stuff users have no control over, and should be pretty rare.

Reply:
It's probably fine as is.

Reply:
I did it this way because we used it similarly in `PlotWindow` for something similar.